CRM/Core/BAO/File.php

```diff
@@ -852,6 +852,8 @@ public static function getEntityTables(): array {
       'civicrm_activity' => ts('Activity'),
       'civicrm_case' => ts('Case'),
       'civicrm_note' => ts('Note'),
+      'civicrm_search_display' => ts('Search Display'),
```

Comment:
I'm still a pretty strong -1 on attaching the file to the display. In order to show all file attachments for each saved search, we really need them to be attached to the search itself. In those rare cases when a search has multiple displays that are all used for file output, they could be differentiated by filename.

Reply:
![Screenshot 2025-06-05 at 3 01 41 PM](https://github.com/user-attachments/assets/2174e6ad-81b3-4b86-948d-2eb915917e6b)